python/modules/connection.py

```diff
@@ -97,8 +97,9 @@ def __init__(self, agent):
         invite_msg = Message({
             '@type': Connection.INVITE,
             'label': self.agent.owner,
-            'key': connection_key,
-            'endpoint': self.agent.endpoint,
+            'recipient_keys': [connection_key],
+            'serviceEndpoint': self.agent.endpoint,
```

Comment:
Would it make sense to use `service_endpoint` instead of `serviceEndpoint` to keep consistent with other field names? Or should we start following camelCase elsewhere?

Reply:
I'm seeing now that `serviceEndpoint` is what the DID Doc uses. I think it makes sense to use the camelCase version but now I'm wondering if we should make all of our attribute names camelCase...